rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1482,6 +1485,13 @@ public long getEstimatedFeesForNextPegOutEvent(Object[] args) throws IOException
         return bridgeSupport.getEstimatedFeesForNextPegOutEvent().value;
     }
 
+    public int setUnionBridgeContractAddressForTestnet(Object[] args) {
+        logger.trace("setUnionBridgeContractAddressForTestnet");
+        // A DataWord cast is used because a SolidityType "address" is decoded using this specific type.
+        RskAddress unionBridgeContractAddress = new RskAddress((DataWord) args[0]);
```

Comment:
Should this be wrapped inside a try/catch? What happens if invalid data is passed?

Reply:
I couldn't recreate this case passing invalid data. When invalid data is passed it stops [here](https://github.com/rsksmart/rskj/blob/61c80377bdd80dc588c0139357e866cbfc9abf45/rskj-core/src/main/java/co/rsk/peg/Bridge.java#L389) , and this is before executing `setUnionBridgeContractAddressForTestnet`.